iguana.corecontroller/src/main/java/org/aksw/iguana/cc/worker/AbstractWorker.java

```diff
@@ -39,282 +35,257 @@
  * and how to test this query.
  *
  * @author f.conrads
- *
  */
 public abstract class AbstractWorker implements Worker {
```

Comment:
Is the separation of Worker and AbstractWorker helpful for anything? Does not necessarily need to be fixed in this PR

Reply:
#195